README.md

```diff
@@ -1 +1,42 @@
-# test-automatiom-testNG
\ No newline at end of file
+# TestNG example
+
+Application model which describes the behavior of running the test with gradle's help
+
+## Getting Started
+
+## Running
+
+1. Open project;
+2. Select the Run Configuration;
+3. Wait.
+
+## Testing
+
+Please use next rules for test the app:
+
+1 -
+``
+SmokeTests or EOSTests
```

Comment:
seems like these rules require additional clarification... try to think about people who haven't seen your library before... would it be easy to understand your thoughts expressed in these 2 sentences?